### PR TITLE
Add lifecycle for an example of data flowing through beginning to end.

### PR DESCRIPTION
**Pull request by gannan08**
***
<a href="https://pr-preview.s3.amazonaws.com/digitalbazaar/vc-data-model/pull/313.html" title="Last updated on Jan 15, 2019, 7:05 PM UTC (5cf0462)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/313/6198a5e...digitalbazaar:5cf0462.html" title="Last updated on Jan 15, 2019, 7:05 PM UTC (5cf0462)">Diff</a>

### CODE REVIEW

**Comment by gannan08** [comment]
@jandrieu @TzviyaSiegman @stonematt this is my attempt at completing the all-encompassing lifecycle example.

**Comment by msporny** [comment]
@gannan08 please fix IPR issues by linking your Github account to your W3C account: https://labs.w3.org/repo-manager/pr/id/w3c/vc-data-model/313

**Comment by TzviyaSiegman** [comment]
Looks good. Thank you

**Comment by stonematt** [comment]
LGTM

File: index.html [hunk 1]
```diff
@@ -643,6 +643,94 @@ <h3>Presentations</h3>
           </figure>
 
     </section>
+    <section>
+      <h3>Lifecycle</h3>
+
+      <em>This section is non-normative.</em>
+
+      <p>
+        After outlining the core data model concepts, it is now possible to
```
[the remark this answers is not in the document]
Comment (gannan08) [reply index.html]: Change Request Addressed Here: https://github.com/w3c/vc-data-model/pull/313/commits/b5156736ac9db3e5ebd5387ade09d4d9131b8f90

File: index.html [hunk 2]
```diff
@@ -643,6 +643,94 @@ <h3>Presentations</h3>
...
+        reach the culmination of this section, the Lifecycle. Claims allow one
+        to make statements about a subject. Credentials provide the composition
+        mechanism for tamper-evident claims that provide cryptographic
+        guarantees on the provenance of those claims. Presentations enable the
+        ability for holders to use one or more credentials to satisfy requests
+        from a verifier and proof of control of those credentials.
+      </p>
+
+      <p>
+        The lifecycle of credentials will take the same general path: issuance
```
[the remark this answers is not in the document]
Comment (gannan08) [reply index.html]: Change Request Addressed Here: https://github.com/w3c/vc-data-model/pull/313/commits/b7b4b7f78bbe330cda771ecd2892874281c815c8

**Comment by gannan08** [comment]
@msporny 
> Thanks for the PR @gannan08! Is this a work in progress or do you want it considered for merging?

PR is finished, please consider for merging.

> @gannan08 please fix IPR issues by linking your Github account to your W3C account: https://labs.w3.org/repo-manager/pr/id/w3c/vc-data-model/313

Should be fixed.

**Comment by David-Chadwick** [comment]
@gannan08 I have the following issues with this PR. Your first paragraph states
"Presentations enable the ability for holders to use one or more credentials to satisfy requests from a verifier and proof of control of those credentials."
whilst  the next paragraph states
" issuance of claims to a credential subject,"
So the first issue is, how did the credential get from the subject to the holder? You should either use the same term in both places i.e. subject, or explain how a subject morphs into a holder who may not be the subject.
Next your example 2 has the subject presenting her VC to a verifier, but there are a couple of problems with this:
a) the ID of the presentation is not the same as the ID of the credential subject, but in your example it is. Both credentials and presentations have IDs, and credentials also have a credentialSubject ID. But presentations do not currently have a holder ID. (I think they should have)
b) your proof has a creator which is key ID. So how does the verifier know that the presenter/holder is the subject and not someone who has stolen the credential from the subject and is presenting it to claim the discount? In other words, what links the creator's key to the credential subject?

Currently I think your informative text raises more issues than clarifications of the data model.

**Comment by msporny** [comment]
> Currently I think your informative text raises more issues than clarifications of the data model.

@gannan08 can you please address @David-Chadwick's concerns, this PR is blocked until those issues are addressed.

**Comment by gannan08** [comment]
@jandrieu Thank you for the clarification.

**Comment by gannan08** [comment]
@msporny Working on it now, should be completed EOB today.

**Pull request closed without merging**